app/controllers/default_controller.rb

```diff
@@ -20,9 +20,10 @@ def status_server
     else
       @status = 500
     end
+    @hostname = Socket.gethostname
     render 'default/status_server',:status => @status
     headers['Last-Modified'] = Time.now.httpdate
-    response.set_header('hostname', Socket.gethostname)
+    response.set_header('hostname', @hostname)
```

Comment:
since this is a custom header, we should prefix with X-

something like X-Status-Check-Hostname